src/Routes/DetailPage/DetailPage.tsx

```diff
@@ -82,26 +84,38 @@ const DetailPage = () => {
     setIsKebabOpen(!isKebabOpen);
   };
 
+  const OnShowConfirmDelete = (_domain: Domain) => {
```

Comment:
The `_domain` param can be removed as it is not used. It will also fix the linter warning.

Reply:
thanks! I have removed the parameter.